docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -37,6 +37,11 @@ To remove the taint added by the command above, you can run:
 kubectl taint nodes node1 key:NoSchedule-
 ```
 
+Additional example, if you want to schedule pods on the master(s), the key will be "node-role.kubernetes.io", so you will run:
```

Comment:
Great example, although I'd recommend a wording change - the sentence is difficult to parse:

> Another example is using the key `node-role.kubernetes.io` to indicate you do not want Pods scheduled on your master nodes. To add this taint to your master node, you can use the command:
```shell
kubectl taint nodes <your_master_node> node-role.kubernetes.io/master:NoSchedule-
```

This makes the wording more direct and uses <> for placeholder values per the style guide (https://kubernetes.io/docs/home/contribute/style-guide/#use-angle-brackets-for-placeholders)